tests/tests/block-stale-reads.sh

```diff
@@ -6,56 +6,21 @@
 # we use triggers to exercise how readers encounter stale blocks.
 #
 
-t_require_commands touch setfattr getfattr
-
-inc_wrap_fs_nr()
-{
-	local nr="$(($1 + 1))"
-
-	if [ "$nr" == "$T_NR_MOUNTS" ]; then
-		nr=0
-	fi
-
-	echo $nr
-}
-
-GETFATTR="getfattr --absolute-names"
-SETFATTR="setfattr"
-
 echo "== create shared test file"
```

Comment:
These little status messages should be updated to say what the new test is doing.   I'd get rid of most of them and just echo that it's using df to issue block reads that trigger the stale invalidation and retry.